frontend/ui/elements/page_turns.lua

```diff
@@ -26,27 +26,57 @@ end
 genTapZonesMenu("default")
 genTapZonesMenu("left_right")
 genTapZonesMenu("top_bottom")
+
+local default_size_b = math.floor(G_defaults:readSetting("DTAP_ZONE_BACKWARD").w * 100)
+local default_size_f = math.floor(G_defaults:readSetting("DTAP_ZONE_FORWARD").w * 100)
+local function readTapZonesSize()
+    local size_b, size_f
+    if G_reader_settings:has("page_turns_tap_zone_forward_size_ratio") then
+        size_f = math.floor(G_reader_settings:readSetting("page_turns_tap_zone_forward_size_ratio") * 100)
+        if G_reader_settings:has("page_turns_tap_zone_backward_size_ratio") then
+            size_b = math.floor(G_reader_settings:readSetting("page_turns_tap_zone_backward_size_ratio") * 100)
+        else
+            size_b = 100 - size_f
+        end
+    else
+        size_b = default_size_b
+        size_f = default_size_f
+    end
+    return size_b, size_f
+end
 table.insert(page_turns_tap_zones_sub_items, {
     text_func = function()
-        local size = math.floor(G_reader_settings:readSetting("page_turns_tap_zone_forward_size_ratio", G_defaults:readSetting("DTAP_ZONE_FORWARD").w) * 100)
-        return T(_("Forward tap zone size: %1%"), size)
+        return T(_("Backward / forward tap zone size: %1 % / %2 %"), readTapZonesSize())
     end,
     enabled_func = function()
         return G_reader_settings:readSetting("page_turns_tap_zones", "default") ~= "default"
     end,
     keep_menu_open = true,
     callback = function(touchmenu_instance)
         local is_left_right = G_reader_settings:readSetting("page_turns_tap_zones") == "left_right"
-        local size = math.floor(G_reader_settings:readSetting("page_turns_tap_zone_forward_size_ratio", G_defaults:readSetting("DTAP_ZONE_FORWARD").w) * 100)
-        UIManager:show(require("ui/widget/spinwidget"):new{
-            title_text = is_left_right and _("Forward tap zone width") or _("Forward tap zone height"),
+        local size_b, size_f = readTapZonesSize()
+        UIManager:show(require("ui/widget/doublespinwidget"):new{
+            title_text = is_left_right and _("Tap zones width") or _("Tap zones height"),
```

Comment:
The plural reads a bit more awkward and it doesn't seem necessary here.
```suggestion
            title_text = is_left_right and _("Tap zone width") or _("Tap zone height"),
```